internal-registry/che-incubator/command-line-terminal/4.5.0/meta.yaml

```diff
@@ -18,7 +18,6 @@ spec:
       attributes:
         protocol: http
         type: ide
-        path: /static/
```

Comment:
wonder if we keep web app on that endpoint... Then probably healthz endpoint should be defined as a separate configuration for endpoint I suppose.
So, please consider creating an issue to extend plugin meta model with health checks.

Reply:
Created https://github.com/eclipse/che/issues/17002